plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/expression/RewriteVarcharConstant.java

```diff
@@ -40,6 +40,9 @@ public Pattern<Constant> getPattern()
     public Optional<String> rewrite(Constant constant, Captures captures, RewriteContext<String> context)
     {
         Slice slice = (Slice) constant.getValue();
+        if (slice == null) {
+            return Optional.empty();
```

Comment:
per https://github.com/trinodb/trino/pull/11396/commits/6c89d7c752d8839e94dab3ff2ec534fd55b6b904#r867723626
at least `RewriteExactNumericConstant` also needs such change (maybe others too)

Reply:
Added, I don't see any other constant rewriters at `io.trino.plugin.jdbc.expression` so I assume these are the only places